lib/utils/net-utils.js

```diff
@@ -4,29 +4,26 @@ const Promise = require('bluebird')
 const net = require('net')
 
 const NetUtils = {
-  isPortAvailable (port, listenAddress) {
-    return new Promise((resolve, reject) => {
-      const server = net.createServer()
+  getAvailablePort (port, listenAddress) {
+    function listen (port, listenAddress) {
```

Comment:
Why we need to create listen function, and invoke it right after ? Can we return new Promise directly in `getAvailablePort` method ?